build.xml

```diff
@@ -978,6 +978,7 @@
           <exclude name="bin/*.in.sh" />
           <include name="tools/bin/*"/>
           <exclude name="tools/bin/*.in.sh" />
+          <include name=".build/docker/*.sh"/>
```

Comment:
what about the `.build/*.sh` scripts ?

Reply:
not sure if it is smart like that, will check

Reply:
@michaelsembwever what you suggested does not work.

Reply:
`.build/**/*.sh` seems to work